python/cuml/cuml/linear_model/mbsgd_classifier.py

```diff
@@ -104,11 +73,7 @@ class MBSGDClassifier(Base, ClassifierMixin, FMajorInputTagMixin):
         Initial learning rate
     power_t : float (default = 0.5)
         The exponent used for calculating the invscaling learning rate
-    learning_rate : {'optimal', 'constant', 'invscaling', 'adaptive'} \
```

Comment:
As noted before, technically a breaking change, but completely fair since the `optimal` option was not supported previously either. So this isn't breaking any user flow.